cmake/libstored.cmake

```diff
@@ -156,6 +156,7 @@ function(libstored_lib libprefix libpath)
 			target_link_libraries(${LIBSTORED_LIB_TARGET} PUBLIC Qt5::Core)
 		else()
 			message(STATUS "Enable Qt6 integration for ${LIBSTORED_LIB_TARGET}")
+            qt_disable_unicode_defines(${LIBSTORED_LIB_TARGET})
```

Comment:
Whether or not this function exists, depends on your Qt version. And the indenting is inconsistent.